lms/djangoapps/lti_provider/management/commands/rescore_lti.py

```diff
@@ -0,0 +1,45 @@
+"""
+Management command to rescore all lti problems for the given course for all users.
+"""
+
+import textwrap
+
+from django.core.management import BaseCommand
+
+from opaque_keys.edx.keys import CourseKey
+
+from lti_provider.models import GradedAssignment
+from lti_provider import tasks
+from student.models import CourseEnrollment
+
+
+class Command(BaseCommand):
+    """
+    Rescore all lti problems for the given courses for all users.
+
+    Examples:
+
+        ./manage.py rescore_lti course-v1:edX+DemoX+Demo_Course course-v1:UBCx+course+2016-01
```

Comment:
Is this docstring accurate? Neither of the following commands work for me locally, and this command isn't listed under `help`:
```
edxapp@precise64:~/edx-platform$ ./manage.py lms --settings=devstack rescore_lti
Unknown command: 'rescore_lti'
Type 'manage.py help' for usage.
edxapp@precise64:~/edx-platform$ ./manage.py --settings=devstack rescore_lti
usage: manage.py [-h] {lms,cms} ...
manage.py: error: invalid choice: 'rescore_lti' (choose from 'lms', 'cms')
edxapp@precise64:~/edx-platform$ ./manage.py lms --settings=devstack help
```